src/BloomExe/Book/BookData.cs

```diff
@@ -714,6 +718,25 @@ private void UpdateDomFromDataSet(DataSet data, string elementName,XmlDocument t
 		}
 
 		/// <summary>
+		/// Fix for invalid HTML markers that look like they're actually email addresses.
+		/// </summary>
+		/// <returns>input string modified as necessary</returns>
+		private static string FixBadlyMarkedEmailAddresses(string s)
+		{
+			// The text we've gathered can contain valid HTML markup, but it can also contain
+			// something like <some_one@somewhere.com> which dies horribly.  How our program
+			// has allowed the latter to occur is beyond me, but let's try to cope with this
+			// situation by encoding the < and > characters.
+			var matches = Regex.Matches(s, @"(<)([-+.\w]+@[-+.\w]+)(>)");
```

Comment:
I don't see why you put parens around the wedges, since you don't use those groups, but I don't suppose it matters much.